_posts/2024-01-19-yannick-horrix-opentofu.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: post
+authors: [yannick-horrix]
+title: 'Choosing Your Orchestrator: OpenTofu vs Terraform'
+image: /img/2024-01-19-yannick-horrix-opentofu/banner.jpg
+tags: [ cloud, automation, terraform, opentofu, iac ]
+category: cloud
+comments: true
+---
+
+> The power of Open Source is the power of the people. The people rule.
+
+
+{:.no_toc}
+- TOC
+{:toc}
+
+# Introduction
+Terraform, an industry-leading infrastructure as code (IaC) tool, has been a cornerstone of cloud provisioning and infrastructure management since its inception in 2014. 
+Over the years, it has fostered a vibrant community comprising thousands of users, contributors, vendors, and an extensive ecosystem of open source modules and extensions.
+
+However, August 10, 2023, marked a significant shift in Terraform’s trajectory when HashiCorp, its stewarding organization, changed the license from the Mozilla Public License (v2.0) to the Business Source License (v1.1). 
+This abrupt change sent ripples across the Terraform community, sparking concerns about its future.
+
+In response, the open source community initiated the OpenTofu project, which was adopted by the Linux Foundation on September 20th. OpenTofu, starting off as a fork from Terraform 1.5.6, aims to create a truly open source alternative to Terraform. 
+
+On January 10, 2024, OpenTofu achieved a pivotal milestone by releasing its first production-stable version, OpenTofu 1.6.0, bringing new features and promising a seamless migration for current Terraform users.
+
+In this article we'll delve into a comparative analysis of OpenTofu and Terraform, exploring the impact of the license change, to help you make an informed decision when choosing the right tool for your infrastructure needs.
+# A Comparative Analysis: OpenTofu vs Terraform
+## Features
+In terms of functionality, both tools are currently equivalent. 
+The OpenTofu community has aimed to maintain feature parity with Terraform for the time being. 
+As such, both currently offer the same large set of features making them well suited tools for IaC workloads.
+
+Some small features were added with the release of OpenTofu 1.6.0 however, including:
```

Comment:
These are changes over the 1.5.6. release (forked version). 
Not different from vanilla TF?